adafruit_connection_manager.py

```diff
@@ -269,82 +259,82 @@ def close_socket(self, socket: SocketType) -> None:
             self._available_sockets.remove(socket)
 
     def free_socket(self, socket: SocketType) -> None:
-        """Mark a managed socket as available so it can be reused."""
+        """Mark a managed socket as available so it can be reused. The socket is not closed."""
         if socket not in self._managed_socket_by_key.values():
             raise RuntimeError("Socket not managed")
         self._available_sockets.add(socket)
 
+    def _register_connected_socket(self, key, socket):
```

Comment:
If willing, I like to group like method types. All the pseudo private methods are up top. Feel free to ignore

Reply:
I will move it up.